src/translations/en-US.json

```diff
@@ -12,12 +12,15 @@
   "h3" : "Punishment",
   "home" : "Home",
   "about" : "About",
+  "whattodo": "What To Do",
   "click" : "Click here to view more",
   "brand" : "Wilson website",
   "example" : "Example heading",
   "hello" : "Hello",
   "counterButton.title" : "Press Here",
   "count" : "Count: {count}",
   "alerthead" : "Oh snap! You got an error!",
-  "specialerror" : "Special Error"
+  "specialerror" : "Special Error",
+  "boring click": "Click me if you bored",
```

Comment:
nit: this should be a key without spaces for the library to handle transform better